Post/views.py

```diff
@@ -7,6 +7,15 @@
 from django.contrib import messages
 
 
+def postListSearch(request):
```

Comment:
Why is it Global function?
Are you going to add tests

Reply:
@Yarboa 
Yes, I will add tests, I just wanted a mid-review about the search.
I forgot to add it under the class(basically there is no need, but I will for the order)